modules/images/webp-uploads/load.php

```diff
@@ -529,17 +529,18 @@ function webp_uploads_img_tag_update_mime_type( $image, $context, $attachment_id
 
 	$basename = wp_basename( $metadata['file'] );
```

Comment:
Actually, there's a conditional at the top of this function: 

https://github.com/WordPress/performance/pull/195/files/76f9392cb7dad591be273cc541ae90137297baf4#diff-71ed188a16b79d179868ee80e76f775462a451e871fc3d6214fcacca3edcea8cR510

This would prevent the execution of the code if the key is not present.

Reply:
Ah right, in that case the code below still needs to be updated though, see other comment.